airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/SqlOperationsUtils.java

```diff
@@ -119,4 +151,14 @@ static void insertRawRecordsInSingleQuery(String insertQueryComponent,
     });
   }
 
+  private static <T> Stream<List<T>> batches(List<T> source, int length) {
```

Comment:
can you add a comment to explain this logic

Reply:
added java doc for this method